templates/server-acl-init-job.yaml

```diff
@@ -96,6 +103,13 @@ spec:
                 {{- if not (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
                 -create-client-token=false \
                 {{- end }}
+                {{- if .Values.global.acls.createReplicationToken }}
+                -create-acl-replication-token=true \
+                {{- end }}
+                {{- if (and .Values.global.acls.replicationToken.secretName .Values.global.acls.replicationToken.secretKey) }}
+                -enable-acl-replication=true \
+                -datacenter={{ .Values.global.datacenter }} \
```

Comment:
This flag is only needed if we're in replication mode. I put it here to make the helm chart backwards compatible with older versions of consul-k8s that don't have this flag. For single-dc users, this flag isn't needed.